utils/src/field_helpers.rs

```diff
@@ -100,6 +106,22 @@ impl<F: Field> FieldHelpers<F> for F {
             bits
         })
     }
+
+    fn from_big(big: BigUint) -> Result<F> {
+        let mut bytes = big.to_bytes_le();
+
+        // Pad with zeros until multiple of 32 if necessary because the BigUint to_bytes_le function gives the smallest possible vector of bytes
+        if bytes.len() % 32 != 0 {
+            bytes.resize(32 + bytes.len() / 32, 0);
```

Comment:
this will also not work if bytes is like >= 2 * 32 of length ?

Reply:
if we really want to write our own (and not use the ones exposed by `PrimeField`) then I think we should just pad:

```rust
if bytes.len() < 32 {
   bytes.extend(std::iter::repeat(0).take(32 - bytes.len());
}
```